dissect/target/helpers/hashutil.py

```diff
@@ -15,13 +15,9 @@
 
 BUFFER_SIZE = 32768
 
-HashRecord = RecordDescriptor(
-    "filesystem/file/digest",
-    [
-        ("path[]", "paths"),
-        ("digest[]", "digests"),
-    ],
-)
+RECORD_NAME = "filesystem/file/digest"
+NAME_SUFFIXES = ["_path", "_resolved_path", "_digest"]
+RECORD_TYPES = ["path", "path", "digest"]
```

Comment:
```suggestion
NAME_SUFFIXES = ["_resolved", "_digest"]
RECORD_TYPES = ["path", "digest"]
```
I don't think we need to duplicate the original path value. As we prepend the field names with the original fieldname, it is easy to see what the original path would have been.